core_backend/app/llm_call/llm_rag.py

```diff
@@ -112,24 +127,20 @@ async def get_llm_rag_answer_with_chat_history(
                 failure_message=RAG_FAILURE_MESSAGE,
                 message_type=message_type,
                 original_language=original_language,
+                original_script=original_script,
             )
         )
-    content = (
-        question
-        + f""""\n\n
-    ADDITIONAL RELEVANT INFORMATION BELOW
-    =====================================
```

Comment:
This would render as a Heading when I display in Markdown, and just thought the HTML syntax is clearer!